lib/src/main/java/com/orange/ods/compose/component/dialog/OdsAlertDialog.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ *
+ *  Copyright 2021 Orange
+ *
+ *  Use of this source code is governed by an MIT-style
+ *  license that can be found in the LICENSE file or at
+ *  https://opensource.org/licenses/MIT.
+ * /
+ */
+
+package com.orange.ods.compose.component.dialog
+
+import androidx.compose.material.AlertDialog
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.window.DialogProperties
+import com.orange.ods.compose.component.button.OdsButtonText
+import com.orange.ods.compose.text.OdsTextBody2
+import com.orange.ods.compose.text.OdsTextSubtitle1
+import com.orange.ods.utilities.extension.ifNotNull
+
+/**
+ * <a href="https://system.design.orange.com/0c1af118d/p/02ae02-dialogs/b/81772e" class="external" target="_blank">ODS alert dialog</a>.
+ *
+ * Alert dialogs interrupt users with urgent information, details, or actions.
+ *
+ * The dialog will position its buttons based on the available space. By default it will try to
+ * place them horizontally next to each other and fallback to horizontal placement if not enough
+ * space is available.
+ *
+ * @param modifier Modifier to be applied to the layout of the dialog.
+ * @param onDismissRequest Executes when the user tries to dismiss the Dialog by clicking outside
+ * or pressing the back button. This is not called when the dismiss button is clicked.
+ * @param confirmButtonText The text of the button which is meant to confirm a proposed action, thus resolving
+ * what triggered the dialog.
+ * @param onConfirmButtonClick The action executed on confirm button click
+ * @param dismissButtonText The text of the button which is meant to dismiss the dialog.
+ * @param onDismissButtonClick The action executed on dismiss button click
+ * @param titleText The title of the Dialog which should specify the purpose of the Dialog. The title
+ * is not mandatory, because there may be sufficient information inside the [text].
+ * @param text The text which presents the details regarding the Dialog's purpose.
+ * @param properties Typically platform specific properties to further configure the dialog.
+ */
+@Composable
+fun OdsAlertDialog(
+    modifier: Modifier = Modifier,
```

Comment:
According to Jetpack Compose guidelines, the `modifier` parameter should appear as the first optional parameter:

> Element functions MUST accept a parameter of type Modifier. This parameter MUST be named “modifier” and MUST appear as the first optional parameter in the element function's parameter list.

Thus I would modify the method signature as follows:

```kotlin
fun OdsAlertDialog(
    text: String,
    confirmButtonText: String,
    onConfirmButtonClick: () -> Unit,
    modifier: Modifier = Modifier,
    onDismissRequest: () -> Unit = {},
    dismissButtonText: String? = null,
    onDismissButtonClick: (() -> Unit)? = null,
    titleText: String? = null,
    properties: DialogProperties = DialogProperties()
)
```

Reply:
Ok thanks, I will follow this rule from now